web/webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const TerserPlugin = require("terser-webpack-plugin");
+
+module.exports = {
```

Comment:
Would also recommend adding hashes to file names:

```js
output: {
        filename: prod ? '[name].[contenthash:8].js' : '[name].js',
        chunkFilename: prod ? '[name].[contenthash:8].js' : '[name].js',
    },
    optimization: {
        usedExports: true,
        moduleIds: 'deterministic',
    },
```

This way, browsers can cache content better